WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -49,7 +51,12 @@
               <% @competition.events.each do |event| %>
                 <td><%= registrations.select { |r| r.events.include?(event) }.length %></td>
               <% end %>
-              <td colspan="4"></td>
+              <% if @competition.guests_enabled? %>
+                <td>
+                  <%= registrations.map { |r| r.guests }.reduce { |sum, guests| sum + guests } %>
```

Comment:
Ruby actually provides syntactic sugar for this, you can change `registrations.map { |r| r.guests }` to `registrations.map(&:guests)`